AVPlayerExample/ViewController.m

```diff
@@ -174,14 +174,38 @@ - (void)startPreview {
         UITapGestureRecognizer *tap = [[UITapGestureRecognizer alloc] initWithTarget:self
                                                                               action:@selector(flipCamera)];
         [self.previewView addGestureRecognizer:tap];
+
+        AVCaptureDevice *frontCamera = [TVICameraSource captureDeviceForPosition:AVCaptureDevicePositionFront];
+
+        if (frontCamera != nil) {
+            [self.camera startCaptureWithDevice:frontCamera completion:^(AVCaptureDevice *device, NSError *error) {
+                if (error != nil) {
+                    [self logMessage:[NSString stringWithFormat:@"Capture failed with error.\ncode = %lu error = %@", error.code, error.localizedDescription]];
+                } else {
+                    self.previewView.mirror = (device.position == AVCaptureDevicePositionFront);
+                }
+            }];
+        }
```

Comment:
What should we do if the front camera is not available? I think there might have been one iPod Touch that we don't actually support like this...

Reply:
Maybe we should assert or otherwise end the program? It seems like a case we were not expecting. See also `flipCamera` below.

Reply:
What ever we decide to do, we also need to do in the Swift code as well. It has the same behavior as this.